src/airules/constants.py

```diff
@@ -0,0 +1,32 @@
+"""Shared constants for explode and implode functionality."""
+
+# Mapping of section headers to their file globs or None for prompts
+# The order of keys determines the order in imploded files
+SECTION_GLOBS = {
```

Comment:
Can we convert this into a JSON file that:

1. We bundle into the CLI and reason
2. We allow the user to override via a CLI argument